gitgud/operations.py

```diff
@@ -215,3 +219,22 @@ def get_operator():
         if os.path.isdir(gg_path):
             return Operator(path)
     return None
+
+def init_tracking_json():
+    if os.path.exists(os.path.join(os.getcwd(), ".git")):
+        with open(".git/gud/commits.json", 'w') as fp:
+            json.dump({}, fp)
+    else:
+        print("ERROR: Wrong directory :C")
+            
+
+def track_commits(commit_num, commit_hash):
+    # Assumes that .git/gud/commits.json has been initialized by 'git gud load'
+    if os.path.exists(".git/gud/commits.json"):
+        with open(".git/gud/commits.json") as fp:
+            commit_dict = json.load(fp)
+        commit_dict["C" + commit_num] = commit_hash
+        with open(".git/gud/commits.json", 'w') as fp:
+            json.dump(commit_dict, fp)
+    else:
+        print("uh oh someting borken")
```

Comment:
For all of these, accept a file operator as a parameter and then use it to get the file paths. If we create a directory and switch to into it (which might be the case with later levels), this won't work

Reply:
done